libnl-route-3.sym

```diff
@@ -204,6 +204,7 @@ global:
 	rtnl_ematch_tree_alloc;
 	rtnl_ematch_tree_dump;
 	rtnl_ematch_tree_free;
+	rtnl_ematch_tree_clone;
```

Comment:
The already release version cannot be changed by adding new symbols to it. New symbols must always be added to the version for the next-coming release (`libnl_3_5` at the moment), at the bottom of the file.